tests/integration/commands/deploy/deploy.test.js

```diff
@@ -817,4 +818,60 @@ describe.skipIf(process.env.NETLIFY_TEST_DISABLE_LIVE === 'true').concurrent('co
       t.expect(response).toEqual('Bundled at deployment')
     })
   })
+
+  test('should upload blobs when saved into .netlify directory', async (t) => {
+    await withSiteBuilder('site-with-blobs', async (builder) => {
+      await builder
+        .withNetlifyToml({
+          config: {
+            build: { publish: 'dist', functions: 'functions' },
+          },
+        })
+        .withContentFile({
+          path: 'dist/.netlify/blobs/deploy/hello',
+          content: 'hello from the blob',
+        })
+        .withContentFile({
```

Comment:
nit: You can use the `.withPackageJson()` helper.

Reply:
I missed that helper! Updated!